common/core/package.json

```diff
@@ -26,7 +26,7 @@
     "alltest": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test*.js",
     "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min && npm -s run check-cover",
     "test": "npm -s run lint && npm -s run build && npm -s run unittest",
-    "check-cover": "istanbul check-coverage --statements 95 --branches 84 --functions 95 --lines 95"
+    "check-cover": "istanbul check-coverage --statements 99 --branches 97 --functions 98 --lines 99"
```

Comment:
nice!

Reply:
Yes.

---
In reply to: [165243974](https://github.com/Azure/azure-iot-sdk-node/pull/217#discussion_r165243974) [](ancestors = 165243974)